tests/bluetooth/bsim_bt/bsim_test_rpa_peripheral/prj.conf

```diff
@@ -0,0 +1,19 @@
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_CENTRAL=y
+
+CONFIG_BT_SMP=y
+
+CONFIG_ASSERT=y
+CONFIG_BT_TESTING=y
+CONFIG_BT_DEBUG_LOG=y
+
+CONFIG_BT_ID_MAX=3
+CONFIG_BT_MAX_PAIRED=2
+
+CONFIG_BT_SMP_ALLOW_UNAUTH_OVERWRITE=y
+CONFIG_BT_ID_UNPAIR_MATCHING_BONDS=y
```

Comment:
do we really need all of those kconfig options? doesn't seem like it. You can try deleting them one by one to get the bare minimum to pass the test.

Reply:
Removed unnecessary options for both tests

Reply:
did you push your changes? I don't see this file changing at all